nuxt-frontend/pages/resources/index.vue

```diff
@@ -3,16 +3,73 @@
     <Header />
     <BlogList
       :mixpanel="$mixpanel"
-      :showDrafts="config.SHOW_DRAFT_POSTS"
-      :showResources="true"
+      :posts="posts"
+      :featurePosts="featurePosts"
+      :count="count"
+      :status="status"
+    />
+    <BlogFooter
+      :mixpanel="$mixpanel"
+      :socialMediaUrl="socialmediadata"
+      :apiUrl="config.STRAPI_URL"
+      :companyName="config.COMPANY_NAME"
     />
-    <BlogFooter :mixpanel="$mixpanel" />
   </div>
 </template>
 
 <script setup>
 import Header from "@/components/partials/NewHeader.vue";
-import config from "./../../config";
+import config from "@/config";
+import { useBlogListStore } from "@/stores/resources";
 
 const { $mixpanel } = useNuxtApp();
+const posts = ref([]);
+const postLimit = ref(10);
```

Comment:
10 should be constant like `const POST_LIMIT = 10` not ref

Reply:
cannot use that as the value needs to be increased as the user reaches the last blog to show more blogs..